daemon/cluster/cluster.go

```diff
@@ -90,6 +90,14 @@ type Config struct {
 	RuntimeRoot string
 }
 
+// NodeAddress consists of related address of a node:
+// Local Address, Listen Address, and Advertise Address
+type NodeAddress struct {
```

Comment:
Thanks @stevvooe. The changes related to this part has already been done in PR #27698. I just removed related code accordingly.